README.md

```diff
@@ -367,7 +367,9 @@ Here's the readme example with configuration options.
 ```python
 import os
 from embedchain import App
-from embedchain.config import InitConfig, AddConfig, QueryConfig
+from embedchain.config.AddConfig import AddConfig
```

Comment:
I agree to your point about not keeping all the imports in `__init__` file and understand its pro/cons as well.

But this pattern is confusing and not simple for the user.

Can we following something like django? https://github.com/django/django/blob/main/django/db/models/__init__.py

```
from django.db import models
```

Reply:
If we do like django, the story remains the same that we will be importing a lot of modules unintentionally. We probably want to give small names to modules to make the path look cleaner to the user going forward.

I will do some refactoring soon to make sure that it is looks elegant to the user.